swerve100/src/main/java/frc/robot/autonomous/DriveToWaypoint2.java

```diff
@@ -184,7 +185,7 @@ public void execute() {
         this.desiredY = desiredState.poseMeters.getY();
 
         // System.out.println("*****************"+goal);
-        var targetChassisSpeeds = m_controller.calculate(m_swerve.getPose(), desiredState, m_swerve, goal.getRotation());
+        var targetChassisSpeeds = m_controller.calculate(m_swerve.getPose(), desiredState, m_gyro, goal.getRotation());
```

Comment:
this value of m_gyro will always be null because you never assigned anything to it.